packages/codemirror/src/editor.ts

```diff
@@ -871,6 +871,210 @@ class CodeMirrorEditor implements CodeEditor.IEditor {
 export
 namespace CodeMirrorEditor {
   /**
+   * The options used to initialize a code mirror editor.
+   */
+  export
+  interface IOptions extends CodeEditor.IOptions {
```

Comment:
Effectively, yes, since we are providing `wordWrap` in this API and not `lineWrapping`.